MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -2,14 +2,16 @@ import Foundation
 import MapboxDirections
 import MapboxNavigationNative
 
-/**
+/** :nodoc:
  A closure to call when the `NavigationDirections` router has been configured completely.
  */
+@available(*, deprecated, message: "Not recommended to use")
```

Comment:
Let’s give no message for these deprecations. Deprecating a symbol already means we don’t recommend using it. In theory, we could say that offline navigation functionality will come back in v1.1, but it’s best not to make forward-looking statements in the API, and chances are the API will look a little different anyways.